code/modules/reagents/reagent_containers/food/snacks.dm

```diff
@@ -607,6 +607,14 @@
 
 	reset_vars_after_duration(resettable_vars, duration)
 
+/obj/item/weapon/reagent_containers/food/snacks/spook()
+	if(reagents.has_reagent(ECTOPLASM))
```

Comment:
it would be better to invert check to avoid rightward creep

not a significant change worth holding this PR up over, however